### PR TITLE
QA_Italian translation

### PR DESCRIPTION
**Pull request by ancap19**

### CODE REVIEW

**Comment by Josexv1** [comment]
Can you mix all the commits into one big commit please.

**Comment by ancap19** [comment]
Not sure how to do that (this is my first github commit). However, if you go to File changed, you should be able to see the whole thing. Sorry for the inconvenience. I'll try to fix it

**Comment by Josexv1** [comment]
Use [this ](https://gist.github.com/patik/b8a9dc5cd356f9f6f980)as a reference to squash all the commits into one.

**Comment by erciccione** [comment]
@ancap19 To fix the commit problems you'll need to squash, i didn't mention that before because in case i can do it for you, but if you want you can do it with git, i can help you get it done. Contact me

File: i18n/it/quality.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
-- Ensure that automation is running on a regular basis
+Il seguente modello riguarda il flusso di lavoro sul controllo di qualità. Posto che tutte le fasi devono essere, in ultima istanza, portate a termine, è possibile lavorare individualmente a ciascuna di esse.
+
+## Prima fase: Revisione di base
+
+- Tutto il codice deve rispettare le linee guida per i collaboratori.
+- Evidenziare le aree che necessitano di un miglioramento, sia esso concettuale o a livello di codice.
+- Creare liste di cose da fare, e, ove possibile, distribuire i compiti.
+
+## Seconda fase: Revisione della specifica /  Implementazione / Annotazione del codice
```
Comment (erciccione) [inline i18n/it/quality.md]: i would change `Annotazione del codice` with `documentazione del codice`

Comment (ancap19) [reply i18n/it/quality.md]: Done. Do I have to change "annotare" with "documentare" few lines later (line 17)?

I'll PM you for the squash :)

Thanks Jose for the suggestion :)

**Comment by erciccione** [comment]
@anonimal :) working with him on this, will be fixed soon

**Pull request merged**